onetimepass/otp.py

```diff
@@ -310,69 +423,42 @@ def get_alias_from_uri(uri: str) -> AliasSchema:
     default=settings.DEFAULT_TIME_STEP_SECONDS,
 )
 @click.option(
-    "counter", "-c", "--counter", type=click.INT, show_default=True, default=0
-)
-@click.option("issuer", "-i", "--issuer")
-@click.option(
-    "otp_type",
+    "initial_time",
     "-t",
-    "--otp-type",
-    type=click.Choice(OTPType),
-    show_default=True,
-    default=OTPType.TOTP.value,
+    "--initial-time",
+    type=click.DateTime(),
+    default=get_default_initial_time,
 )
 @click.pass_context
-def add(
+def add_totp(
```

Comment:
I would suggest adding `help="Add the new TOTP secret as the specified ALIAS."`, WDYT?

Reply:
done